src/components/RightPane/CoursePane/SchoolDeptCard.tsx

```diff
@@ -1,8 +1,10 @@
 import React, { PureComponent } from 'react';
-import { Grid, Paper, Typography, withStyles, Collapse } from '@material-ui/core';
+import {Grid, Paper, Typography, withStyles, Collapse, Theme} from '@material-ui/core';
 import { ExpandLess, ExpandMore } from '@material-ui/icons';
+import {ClassNameMap} from "notistack";
+import {Styles} from "@material-ui/core/styles/withStyles";
 
-const styles = (theme) => ({
+const styles = (theme: Theme) => ({
```

Comment:
Not sure what you mean because TypeScript balks about `theme` being implicitly typed `any` when I remove the type parameter.

Reply:
Maybe I got the terminology wrong, but by type parameters, I meant the stuff that goes in angle brackets, like `Theme` and `object` in `Styles<Theme, object>`. Now I'm realizing that this is Theme instead of Style and it's on the parameter of `styles` instead of `styles` itself, which is why it didn't need any parameters. This way seems simpler, so can you just quickly replace all instances of `const styles: Styles<Theme, object> = (theme)` with `const styles = (theme: Theme)` and remove any unused `Styles` imports?

Reply:
Now that I've looked at it, this is actually an error on my part since the return value of the following statement:

```
const styles = (theme: Theme) => ({ […] });
```

is implicitly `any`, and therefore in order for things to work properly I need to add the ugly `as unknown as Styles<Theme, {}>` hack that we want to avoid.

Whereas with your approach:

```
const styles: Styles<Theme, object> = (theme) => ({ […] });
```

the type of the `theme` parameter is correctly inferred to be `Theme`, probably because of the `withStyles()` call at the very end of the file.

So what I think is the right thing to do instead is to replace all instances of the former with the latter, and also remove the double typecast that we wanted to get rid of anyways.